jme3-core/src/main/java/com/jme3/anim/AnimComposer.java

```diff
@@ -66,7 +66,9 @@ public class AnimComposer extends AbstractControl {
      * Instantiate a composer with a single layer, no actions, and no clips.
      */
     public AnimComposer() {
-        layers.put(DEFAULT_LAYER, new AnimLayer(this, DEFAULT_LAYER, null));
+        if(layers.isEmpty()) {
+            layers.put(DEFAULT_LAYER, new AnimLayer(DEFAULT_LAYER, null));
+        }
```

Comment:
Not sure if this is needed, but feels safer in case someone deletes it and saves